src/test/e2e/stores/api-token-store.e2e.test.ts

```diff
@@ -35,3 +39,146 @@ test('get token returns the token when exists', async () => {
     expect(foundToken.tokenName).toBe(newToken.tokenName);
     expect(foundToken.type).toBe(newToken.type);
 });
+
+describe('count deprecated tokens', () => {
+    test('should return 0 if there is no legacy or orphaned tokens', async () => {
+        await stores.projectStore.create({
+            id: 'test',
+            name: 'test',
+        });
+        await stores.apiTokenStore.insert({
+            secret: '*:*.be44368985f7fb3237c584ef86f3d6bdada42ddbd63a019d26955178',
```

Comment:
Almost all fields can count - for example tokens of ADMIN type can't be orphaned, because it doesn't have projects. At the same time it can be in v1 or v2 format. I'm providing a variety of possible _correct_ tokens.